sdk/monitor/azquery/example_test.go

```diff
@@ -0,0 +1,81 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azquery_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/monitor/azquery"
+)
+
+var cred *azidentity.DefaultAzureCredential
+var kustoQuery1 string
+var kustoQuery2 string
+var kustoQuery3 string
+var workspaceID string
+var resourceURI string
+
+func ExampleNewLogsClient() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		// TODO: handle error
+	}
+
+	client := azquery.NewLogsClient(cred, nil)
+	_ = client
+}
+
+func ExampleNewMetricsClient() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		// TODO: handle error
+	}
+
+	client := azquery.NewMetricsClient(cred, nil)
+	_ = client
+}
+
+func ExampleLogsClient_QueryWorkspace() {
+	client := azquery.NewLogsClient(cred, nil)
+	timespan := azquery.TimeInterval(time.Now().Add(-12*time.Hour), time.Now())
+
+	res, err := client.QueryWorkspace(context.TODO(), workspaceID,
+		azquery.Body{Query: to.Ptr(query), Timespan: to.Ptr(timespan)}, nil)
+	if err != nil {
+		// TODO: handle error
+	}
+	_ = res
```

Comment:
It can be nice to show some usage here, especially for common fields.

Reply:
plan on adding to documentation